RealmTasks Apple/RealmTasks macOS/NSView+Animations.swift

```diff
@@ -20,13 +20,48 @@ import Cocoa
 
 extension NSView {
 
-    static func animateWithDuration(duration: NSTimeInterval, animations: () -> Void, completion: (() -> Void)? = nil) {
+    static var defaultAnimationDuration: NSTimeInterval {
+        return 0.2
+    }
+
+    static var defaultAnimationTimingFunction: CAMediaTimingFunction {
+        return .easeInEaseOut()
+    }
+
+    static func animate(duration duration: NSTimeInterval = defaultAnimationDuration, timingFunction: CAMediaTimingFunction = defaultAnimationTimingFunction, animations: () -> Void, completion: (() -> Void)? = nil) {
         NSAnimationContext.runAnimationGroup({ context in
-            context.duration = duration
             context.allowsImplicitAnimation = true
 
+            context.duration = duration
+            context.timingFunction = timingFunction
+
             animations()
         }, completionHandler: completion)
     }
 
+    // Convenient method for trailing closure syntax without completion handler
```

Comment:
why is this necessary given that the other one has a `completion` default value of `nil`?